EIPS/eip-894.md

```diff
@@ -0,0 +1,39 @@
+## Preamble
+
+    EIP: 894
+    Title: Immutability Enforcement Proposal (IMP)
+    Author: Marius Kjærstad
+    Type: Meta
+    Status: Draft
+    Created: 2018-02-21
+
+
+## Definition
+The definition of bailouts in this EIP is state changes to the Ethereum blockchain, through a hardfork, that treat account holders in different ways.
```

Comment:
Every transaction is a state change, so you probably wanted to say 'irregular state changes'. And define what is an 'irregular state change'.

Reply:
It says "through a hardfork" and regular transactions do not happen by hardforks.